CodePushPackage.m

```diff
@@ -285,6 +285,11 @@ + (void)downloadPackage:(NSDictionary *)updatePackage
                 [CodePushPackage copyEntriesInFolder:unzippedFolderPath
                                           destFolder:newPackageFolderPath
                                                error:&error];
+                if (error) {
```

Comment:
Was this added for good measure or is there a scenario you thought of that this could cause problems with?